source/code/providers/SCX_FileSystemStatisticalInformation_Class_Provider.cpp

```diff
@@ -51,6 +51,7 @@ static void EnumerateOneInstance(
 
         scxulong data1;
         scxulong data2;
+        scxulong data3;
```

Comment:
I suggest we can rename the variable names to mbUsed, mbFree, mbTotal.

Reply:
Let it be like this only, It can be use for multiple purposes.

Reply:
ok